edx_sga/static/js/src/edx_sga.js

```diff
@@ -48,42 +53,42 @@ function StaffGradedAssignmentXBlock(runtime, element) {
                 },
                 progressall: function(e, data) {
                     var percent = parseInt(data.loaded / data.total * 100, 10);
-                    $(content).find(".upload").text(
-                        "Uploading... " + percent + "%");
+                    $(content).find('.upload').text(
+                        'Uploading... ' + percent + '%');
                 },
                 fail: function(e, data) {
                     /**
-                     * Nginx and other sanely implemented servers return a 
-                     * "413 Request entity too large" status code if an 
+                     * Nginx and other sanely implemented servers return a
+                     * "413 Request entity too large" status code if an
                      * upload exceeds its limit.  See the 'done' handler for
                      * the not sane way that Django handles the same thing.
                      */
-                    if (data.jqXHR.status == 413) {
+                    if (data.jqXHR.status === 413) {
                         /* I guess we have no way of knowing what the limit is
                          * here, so no good way to inform the user of what the
                          * limit is.
                          */
-                        state.error = "The file you are trying to upload is too large."
+                        state.error = 'The file you are trying to upload is too large.';
                     }
                     else {
```

Comment:
Even though `gjslint` isn't catching it, another helpful thing in this PR would be to format curly braces in the style mentioned here: https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml#Code_formatting

Notice that curly braces in this file are formatted like:
```javascript
if (condition) {
    // execute if condition
}
else {
    // execute if not condition
}
```

Where the style guide recommends:
```javascript
```javascript
if (condition) {
    // execute if condition
} else {
    // execute if not condition
}
```
```